From 1c21e19ae3847f7ca25d466c5af04d0137a28f36 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Tue, 1 Apr 2014 08:48:09 -0400
Subject: [PATCH 16/41] Actually do a fix for rhbz#1082318 that fixes the issue
 there.

News: after all these years I'm still terrible at shell scripts, just
like everybody else.

Related: rhbz#1063534

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 new-kernel-pkg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/new-kernel-pkg b/new-kernel-pkg
index 034fa3b..d9a9b67 100755
--- a/new-kernel-pkg
+++ b/new-kernel-pkg
@@ -153,7 +153,7 @@ install() {
     fi
 
     DEVTREE=""
-    if [ "x$devtreefile" != "x" -a -f $devtreefile ]; then
+    if [ "x$devtreefile" != "x" -a -f "$devtreefile" ]; then
 	[ -n "$verbose" ] && echo "found $devtreefile and using it with grubby"
 	DEVTREE="--devtree $devtreefile"
     fi
-- 
2.4.3

